tests/integration/standard/test_control_connection.py

```diff
@@ -101,7 +101,7 @@ def test_get_control_connection_host(self):
         self.assertNotEqual(host, new_host)
 
     # TODO: enable after https://github.com/scylladb/python-driver/issues/121 is fixed
-    @unittest.skip('Fails on scylla due to the broadcast_rpc_port is None')
+    @pytest.mark.skip('Fails on scylla due to the broadcast_rpc_port is None')
```

Comment:
What is the difference, why did you replace it ?

Reply:
as mentioned in the commit message, unittest.skip doesn't skip the setup/teardown code when using pytest as test runner, hence creating the cluster even for skipped tests.

for some of those test it was an issue, since the cassandra couldn't create the cluster.